### DISCUSSION TITLE
Quicksync starting at 01/0/1/2021; or fetch data from mempool separately

### DISCUSSION THREAD

**Discussion by sigmal3** [cryptoadvance/specter-desktop#1266] [General]
Hello 

my internet connection and hardware are both limited, I cannot download and process the full node.
The quicksync is too recent for my use and do not display older transactions.
Is there any possibility to have an option with a quicksync starting at 01/01/2021?
Or maybe a way to manually and separately insert/fetch data from mempool, instead of all at once to preserve privacy? for example 1 by 1, every couple of days?

thank you for your powerful program

**Comment by k9ert** [comment]
What's wrong with fetching all at once if you run that via Tor? Another possibility would be to buy a raspi and run a fullnode. I think we already provide a variety of options and i can't see much value in adding this.
